lib/jekyll-heroicons.rb

```diff
@@ -63,13 +63,7 @@ def prepare(markup)
 
     # This method checks if heroicons liquid tag has a variant specified. If not, it check default variant in the config file. If both of those are not defined, it falls back to the default variant.
     def variant(markup)
-      if @options.key?(:variant)
-        @options[:variant]
-      elsif config["variant"]
-        config["variant"]
-      else
-        DEFAULT_VARIANT
-      end
+      @options[:variant] || config["variant"] || DEFAULT_VARIANT
```

Comment:
Can you try this code?

```suggestion
      @options&.dig(:variant) || config&.dig("variant") || DEFAULT_VARIANT
```